README.md

```diff
@@ -32,7 +32,7 @@ defmodule GitHub do
   plug Tesla.Middleware.JSON
 
   def user_repos(login) do
-    get("/users/" <> login <> "/repos")
+    get(login <> "/repos")
```

Comment:
The original version seems correct, 
`https://api.github.com/users/teamon/repos` returns expected response while the one without `users` `https://api.github.com/teamon/repos` returns 404

Reply:
This will break, the endpoint is `https://api.github.com/users/{user handler}/repos`